### PR TITLE
feat(admin): document the `--showonly` flag of the app update command

### PR DESCRIPTION
**Pull request by SimJoSt**
### 🖼️ Screenshots

Screenshot 2023-12-14 at 01 05 12

### CODE REVIEW

**Comment by SimJoSt** [comment]
Source of the information of the existence of this flag: https://help.nextcloud.com/t/check-for-app-updates/83403/4

**Review by tflidd — APPROVED** [review]
Code was added a long time ago: https://github.com/nextcloud/server/pull/11053
and it is a follow up on this https://github.com/nextcloud/documentation/pull/1653

File: admin_manual/configuration_server/occ_command.rst [hunk 1]
```diff
@@ -264,6 +264,10 @@ To update all apps::
 
     sudo -u www-data php occ app:update --all
 
+To show only available updates::
```
Comment (joshtrichards) [inline admin_manual/configuration_server/occ_command.rst:267]:
```suggestion
To show available update(s) without updating::
```

**Review by joshtrichards — APPROVED** [review]

**Review by SimJoSt — COMMENTED** [review]
It sounds better after the change. I was trying to keep the phrasing similar to the name of the flag 😄

**Pull request merged**